packages/hurumap-next/eslint.config.js

```diff
@@ -0,0 +1,34 @@
+const eslintConfig = require("eslint-config-commons-ui/next");
+
+module.exports = [
+  {
+    ignores: [
+      "**/node_modules",
+      "**/.pnp",
+      "**/.pnp.js",
+      "**/.pnpm-debug.log",
+      "**/dist/",
+      "**/coverage",
+      "**/.next/",
+      "**/out/",
+      "**/build/",
+      "**/.DS_Store",
+      "**/*.pem",
+      "**/npm-debug.log*",
+      "**/yarn-debug.log*",
+      "**/yarn-error.log*",
+      "**/.vercel",
+      "**/.now",
+      "**/.turbo",
+      "**/test-results/",
+      "**/playwright-report/",
+    ],
+  },
```

Comment:
Can't we defined all these ignores in either `eslint-config-commons-ui/core` or `eslint-config-commons-ui/next` and just reuse them in other packages/apps?